Source/ToolCore/NETTools/NETProjectSystem.cpp

```diff
@@ -214,6 +217,7 @@ namespace ToolCore
 
             if (build)
             {
+                build->UnsubscribeFromEvent(E_NETBUILDRESULT); //unsubscribe in case it has been subscribed before
                 build->SubscribeToEvent(E_NETBUILDRESULT, ATOMIC_HANDLER(NETProjectSystem, HandleNETBuildResult));
```

Comment:
ok, so this isn't wxWidgets, where you can hang multiple event handlers on an event.